src/components/Contentful/index.js

```diff
@@ -1,74 +1,78 @@
+  const id = content.name
+    .toLowerCase()
+    .replace(/\s/g, '')
+    .replace('>', '-')
 
   switch (content.__typename) {
-  case 'ContentfulInlineImages':
-    return (
-      <InlineImages
-        caption={content.caption}
-        columnWidth={content.columnWidth}
-        columnOffset={content.columnOffset}
-        constrainImageHeight={content.constrainImageHeight}
-        id={id}
-        key={i}
-        images={content.images}
+    case 'ContentfulInlineImages':
+      return (
+        <InlineImages
+          caption={content.caption}
+          columnWidth={content.columnWidth}
+          columnOffset={content.columnOffset}
+          constrainImageHeight={content.constrainImageHeight}
+          id={id}
+          key={i}
+          images={content.images}
         />
-    )
-  case 'ContentfulJumbotron':
-    return (
-      <Jumbotron
-        backgroundUrl={content.background && content.background.fixed.src}
-        columnWidth={content.columnWidth}
-        columnOffset={content.columnOffset}
-        html={content.childContentfulJumbotronBodyRichTextNode.childContentfulRichText.html}
-        id={id}
-        key={i}
-        textAlign={content.textAlign}
+      )
+    case 'ContentfulJumbotron':
+      return (
+        <Jumbotron
+          backgroundUrl={content.background && content.background.fixed.src}
+          columnWidth={content.columnWidth}
+          columnOffset={content.columnOffset}
+          html={
+            content.childContentfulJumbotronBodyRichTextNode
+              .childContentfulRichText.html
+          }
+          id={id}
+          key={i}
+          textAlign={content.textAlign}
         />
-    )
-  case 'ContentfulProse':
-    return (
-      <Prose
-        columnWidth={content.columnWidth}
-        columnOffset={content.columnOffset}
-        html={content.childContentfulProseBodyRichTextNode.childContentfulRichText.html}
-        id={id}
-        key={i}
-        textAlign={content.textAlign}
+      )
+    case 'ContentfulProse':
+      return (
+        <Prose
+          columnWidth={content.columnWidth}
+          columnOffset={content.columnOffset}
+          html={
+            content.childContentfulProseBodyRichTextNode.childContentfulRichText
+              .html
+          }
+          id={id}
+          key={i}
+          textAlign={content.textAlign}
         />
-    )
-  case 'ContentfulConvertFlowArea':
-    return (
-      <ConvertFlowArea
-         convertFlowDivClass={content.convertFlowDivClass}
-         id={id}
-         key={i}
-         />
-    )
-  case 'ContentfulGrid':
-    return (
-      <Grid
-         alignItems={content.alignItems}
-         content={content.content}
-         id={id}
-         key={i}
-         style={content.style}
-         />
-    )
-  default:
-    return <div>Unknown Content Type: {content.__typename}</div>
+      )
+    case 'ContentfulConvertFlowArea':
+      return <HubSpotArea id={id} key={i} />
```

Comment:
/Slimed. But if we can't use Contentful (see below), how to we trigger this case statement?